PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -173,7 +170,11 @@ extension HomeViewModel {
                 )
             }
         }
-
+        
+        if snapshot.numberOfSections == 0 {
+            return Self.buildLoadingSnapshot()
+        }
```

Comment:
is there a case where we don't want to show the loading cell if there are no items?